instrumentation/active_record/lib/opentelemetry/instrumentation/active_record/patches/querying.rb

```diff
@@ -18,7 +18,7 @@ class << base
 
           # Contains ActiveRecord::Querying to be patched
           module ClassMethods
-            def find_by_sql(sql, binds = [], preparable: nil, &block)
+            def find_by_sql(...)
```

Comment:
Are there other methods here that need the same treatment?

Reply:
No it does not look like it.